src/ui/render.ts

```diff
@@ -22,6 +25,7 @@ export default class UI {
   selectedHeaderCell: HTMLElement | null = null;
   selectedCellsContainer: SelectionContainer;
   isReadOnly: boolean;
+  latestSelectedCell: { col: number; row: number } | undefined;
```

Comment:
I believe we can just name this selectedCell.

Reply:
and make a type out of this  { col: number; row: number } in the types file

Reply:
Isn't the existing `Coordinate` type sufficient?

Reply:
I think Coordinate type is sufficient